.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please remove these files, feel free to add `/.idea/` or similar to `.gitignore`.

Reply:
removed .idea dir